### DISCUSSION TITLE
3400！

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3400] [讨论]
短短一个月就从3300突破到3400了，这该悲哀呢还是高兴呢……

**Comment by Deep-Dark-Forest** [comment]
当然应该高兴了，毕竟我们现在有了两个新的超级活跃的协作者再多也不怕

**Reply by allMagicNB** [reply]
![d7fe340b34f582f65e9c363857564f78](https://github.com/Hex-Dragon/PCL2/assets/116283787/9cb6bff6-cbfd-4a71-bd58-385c96325725)

**Reply by MoYuan-CN** [reply]
好好好

**Reply by Ad-closeNN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/0cc99769-52b9-4cf0-b0bc-362a7d64a2e7)
活 跃 黑（确信

**Reply by MoYuan-CN** [reply]
哈哈哈哈

**Reply by Deep-Dark-Forest** [reply]
那凌云是啥？活跃灰？活跃粉？活跃白？活跃粽？

**Reply by MoYuan-CN** [reply]
活跃云（笑

**Reply by Ad-closeNN** [reply]
这称号不错（

**Reply by Deep-Dark-Forest** [reply]
![Screenshot_20240224_164418_com.huawei.browser_edit_903650849298040.jpg](https://github.com/Hex-Dragon/PCL2/assets/131975043/4239b50e-dcf9-46b4-859b-a862fd9f880f)

淘气黑（确信）

P.S.他填的方块都是tnt

**Comment by Ad-closeNN** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/52a18dba-c372-40a9-8532-bde5fab9025d)
升级，把我主题抠没了（

**Reply by Ad-closeNN** [reply]
我正版信息也没了...

**Reply by Ad-closeNN** [reply]
谢谢龙猫（悲
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/6a89aefe-3f3b-4853-a571-f65b2410753c)

**Reply by MoYuan-CN** [reply]
你设备信息变了？

**Reply by Ad-closeNN** [reply]
没...PCL升级就没了，你可以看看#3401，那个时候我主题还是对的
EDIT:留了后手(但不多 
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/bd5c38b9-e585-42bf-a1b7-ac98978ac409)

**Reply by Ad-closeNN** [reply]
回档了还是不行...

**Reply by MoYuan-CN** [reply]
主题是不能用这个回档的..

**Reply by SqrtTan** [reply]
~你这什么神仙启动器（指左上角文字）~

**Reply by hejiehao** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/898e350e-4028-49b7-95ae-e3661388dcc1)
